packages/@aws-cdk/aws-scheduler-alpha/lib/group.ts

```diff
@@ -0,0 +1,372 @@
+   *
+   * @default - sum over 5 minutes
+   */
+  metricSentToDLQTrunacted(props?: cloudwatch.MetricOptions): cloudwatch.Metric {
+    return this.metric('InvocationsSentToDeadLetterCount_Truncated_MessageSizeExceeded', props);
+  }
+
+  /**
+   * Grant the indicated permissions on this group to the given principal
+   */
+  grant(grantee: iam.IGrantable, ...actions: string[]): iam.Grant {
+    return iam.Grant.addToPrincipal({
+      grantee,
+      actions,
+      resourceArns: [this.groupArn],
+      scope: this,
+    });
+  }
+
+  private arnForScheduleInGroup(scheduleName: string): string {
+    return Arn.format({
+      region: this.env.region,
+      account: this.env.account,
+      partition: Aws.PARTITION,
+      service: 'scheduler',
+      resource: 'schedule',
+      resourceName: this.groupName + '/' + scheduleName,
+    });
+  }
+
+  /**
+   * Grant list and get schedule permissions for schedules in this group to the given principal
+   */
+  grantReadSchedules(identity: iam.IGrantable) {
+    return iam.Grant.addToPrincipal({
+      grantee: identity,
+      actions: ['scheduler:GetSchedule', 'scheduler:ListSchedules'],
+      resourceArns: [this.arnForScheduleInGroup('*')],
+      scope: this,
+    });
+  }
+
+  /**
+   * Grant create and update schedule permissions for schedules in this group to the given principal
+   */
+  grantWriteSchedules(identity: iam.IGrantable): iam.Grant {
+    return iam.Grant.addToPrincipal({
+      grantee: identity,
+      actions: ['scheduler:CreateSchedule', 'scheduler:UpdateSchedule'],
+      resourceArns: [this.arnForScheduleInGroup('*')],
+      scope: this,
+    });
+  }
+
+  /**
+   * Grant delete schedule permission for schedules in this group to the given principal
+   */
+  grantDeleteSchedules(identity: iam.IGrantable): iam.Grant {
+    return iam.Grant.addToPrincipal({
+      grantee: identity,
+      actions: ['scheduler:DeleteSchedule'],
+      resourceArns: [this.arnForScheduleInGroup('*')],
+      scope: this,
+    });
+  }
+}
+
+export class Group extends GroupBase {
+  /**
+   * Import an external group by ARN.
+   *
+   * @param scope construct scope
+   * @param id construct id
+   * @param groupArn the ARN of the group to import (e.g. `arn:aws:scheduler:region:account-id:schedule-group/group-name`)
+   */
+  static fromGroupArn(scope: Construct, id: string, groupArn: string): IGroup {
+    const arnComponents = Stack.of(scope).splitArn(groupArn, ArnFormat.SLASH_RESOURCE_NAME);
+    const groupName = arnComponents.resourceName!;
+    class Import extends GroupBase {
+      groupName = groupName;
+      groupArn = groupArn;
+    }
+    return new Import(scope, id);
+  }
+
+  /**
+   * Import a default schedule group.
+   *
+   * @param scope construct scope
+   * @param id construct id
+   */
+  static fromDefaultGroup(scope: Construct, id: string): IGroup {
+    return Group.fromGroupName(scope, id, 'default');
+  }
```

Comment:
I think this is a bit of a misnomer. IIRC we are not importing a default schedule group, we are referencing the default schedule group that we know will be created alongside the schedule?

If so, I suggest naming this function `defaultGroup` so it can be used as `Group.defaultGroup`. Now that I think about it, does this method belong here at all?

It feels like the use case is to attach a default group onto a scheduler if no group is provided:

```ts
this.group = props.group ?? Group.fromDefaultGroup(this, "DefaultGroup");
```

Is there a case to expose this as a static property at all? Or should it just be an internal method for `Schedule` to be called if no group is provided?

Reply:
> Now that I think about it, does this method belong here at all?

Yeah, I think there are the following use-cases of using the default group: 

1. Get metrics
2. Grant permissions (to the schedules inside group and to the group itself)
3. Explicitly add schedules to the default group 

Do not mind calling `Group.defaultGroup`. I just thought that if we pass a `scope` to the method, it should have the word `from`, because we will be creating a CDK resource (but not a CFN resource)

Reply:
I think the `from` keyword implies that we are grabbing a resource "from" somewhere that already exists in your aws account. But I'm not married to the semantics, I don't think its important either way.

Sounds like there is a use case beyond what I see in the implementation. Can we add a section to the README that explains how to access the defaultGroup then?

Reply:
> I think the from keyword implies that we are grabbing a resource "from" somewhere that already exists in your aws account. But I'm not married to the semantics, I don't think its important either way.

Exactly. That's why we used `fromDefaultGroup` as default group already exists in our AWS accounts (it doesn't need to be created). 

> Can we add a section to the README that explains how to access the defaultGroup then?

Will do, good call out.